source/get.d.ts

```diff
@@ -1,5 +1,6 @@
 import {StringDigit} from '../source/utilities';
 import {Split} from './split';
+import {StringKeyOf} from './string-key-of.js';
```

Comment:
```suggestion
import {StringKeyOf} from './string-key-of';
```

The `.js` extension is only required when a package is ESM.